apps/server/src/repositories/learnroom/dashboard.model.mapper.ts

```diff
@@ -0,0 +1,44 @@
+import { Collection, wrap } from '@mikro-orm/core';
+import {
+	DashboardEntity,
+	GridElement,
+	IGridElement,
+	DefaultGridReference,
+} from '../../entities/learnroom/dashboard.entity';
+import { DashboardGridElementModel, DashboardModelEntity, DefaultGridReferenceModel } from './dashboard.model.entity';
+
+export class DashboardModelMapper {
+	static mapGridElementToModel(gridElement: IGridElement, dashboard: DashboardModelEntity): DashboardGridElementModel {
```

Comment:
it definetly wouldnt hurt.
So far I didnt do it because I didnt feel I needed to, becasue the "mapToEntity" function is pretty straight forward even without helper function (whereas mapToModel got pretty involved without a helper).

But it would definetly be a good thing to refactor that, and it is solely your disgession to enforce it before merging :)

Reply:
Aha, so it is mostly for internal use. Externally we are only interested in the mapToModel and mapToEntity. Then, I agree, not needed at the moment.